build/tools/PrepareRelease/GenerateIntegrationDefinitions.cs

```diff
@@ -19,6 +19,59 @@ public static void Run(params string[] outputDirectories)
             var assemblies = new List<Assembly>();
             assemblies.Add(typeof(Instrumentation).Assembly);
 
+            // find all methods in Datadog.Trace.ClrProfiler.Managed.dll with [InstrumentMethod]
+            // and create objects that will generate correct JSON schema
+            var callTargetIntegrations = from assembly in assemblies
+                                         from wrapperType in assembly.GetTypes()
+                                         let attributes = wrapperType.GetCustomAttributes<InstrumentMethodAttribute>(inherit: false)
+                                         where attributes.Any()
+                                         from attribute in attributes
+                                         let integrationName = GetIntegrationName(wrapperType)
```

Comment:
For integrations like AdoNet where we have multiple methods to wrap (like `ExecuteReader`, `ExecuteScalar`, `ExecuteReaderAsync`, etc.) they all need to have the same integration name. We previously had a separate `Integration` property on the attribute. Do you think we'll need to add that back in or do you have a different idea for unifying the integration names?

Reply:
what's the use of IntegrationName in the json? to enable/disable that integration as a group? If is used for that I think we can put it in the attribute.

Reply:
Yes, it's used to enable/disable the integration as a group

Reply:
Ok, so I'll add that to the attribute.

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1007/commits/32df4ab8775a252e6a66634308b2957cb487dcc6